sdk/ml/azure-ai-ml/azure/ai/ml/_internal/_schema/environment.py

```diff
@@ -19,3 +19,9 @@ class InternalEnvironmentSchema(PathAwareSchema):
     name = fields.Str()
     version = fields.Str()
```

Comment:
why not use the core  schema VersionField?

Reply:
Change to VersionField